HHS/Head-Start-TTADP
# [TTAHUB-627] Add Filter for Other Entities

## Description of change

Added filter for Other Entities with type ahead.

## How to test

Go to the AR landing page and add a filter for Other Entities. AR's with this entity should only be shown.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-627


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
